market_next_door_api/market_next_door_api/models.py

```diff
@@ -42,18 +55,6 @@ class Item(models.Model):
     def __str__(self):
         return self.item_name
 
-class Market(models.Model):
-    market_name = models.CharField(max_length=50)
-    location = models.CharField(max_length =100)
-    details = models.CharField(max_length=100)
-    startdate = models.DateField(null=True)
-    enddate = models.DateField(null=True)
-    date_created = models.DateTimeField(auto_now_add=True)
-    updated_at = models.DateTimeField(auto_now=True)
-
-    def __str__(self):
-        return self.market_name
-
```

Comment:
It turns out that to create a ForeignKey or ManyToManyField associated to the model you want, the model you reference needs to exist ahead of the code in which you call it, which is why I moved it to the top.